src/main/java/com/example/bankProjetc/controller/TransationController.java

```diff
@@ -0,0 +1,56 @@
+package com.example.bankProjetc.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.bankProjetc.Views.Views;
+import com.example.bankProjetc.model.Transation;
+import com.example.bankProjetc.services.TransationService;
+import com.fasterxml.jackson.annotation.JsonView;
+
+@RestController
+public class TransationController {
+	@Autowired
+	TransationService transatioService;
+
+	@PostMapping("/transaction")
+	public ResponseEntity<Transation> createTransation(@RequestBody @JsonView(value = Views.TransactionView.post.class) Transation transation) {
```

Comment:
Do it like below everywhere

@PostMapping("/transaction")
@JsonView(value = Views.TransactionView.post.class)
public ResponseEntity<Transation> createTransation(@RequestBody Transation transation) {